### PR TITLE
Icon config

### PR DESCRIPTION
**Pull request by rofe**
Proposal to enable #523 without breaking backward compatibility.

### CODE REVIEW

File: src/schemas/secrets.schema.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "meta:license": [
-    "Copyright 2018 Adobe. All rights reserved.",
+    "Copyright 2019 Adobe. All rights reserved.",
```
Comment (trieloff) [inline src/schemas/secrets.schema.json]: I don't think you have to update the year here. The initial claim has been made in 2018, what has been added in 2019 is only incremental.

Comment (rofe) [reply src/schemas/secrets.schema.json]: OK

**Comment by rofe** [comment]
> The code looks good, but do we really _need_ the config?

If we want to enable #523 without disabling single SVG icon files at the same time, I thought a config would be the least intrusive/breaking way to have both. But I'm open for alternative solutions.

**Comment by trieloff** [comment]
Ok

**Comment by rofe** [comment]
I agree that we'll need a solution that works for helix-pages. Closing this PR.

**Pull request closed without merging**